microsoft-edge/webview2/releasenotes.md

```diff
@@ -16,6 +16,51 @@ The WebView2 team updates the [WebView2 SDK][NuGetGallery] on a six-week cadence
 > [!NOTE]
 > Re-compile your app after updating the NuGet package.  
 
+## 1.0.774-prerelease  
+
+Release Date: January 25, 2021  
+
+[NuGet package][NuGetGallery1.0.774-prerelease] \| minimum Microsoft Edge version 86.0.616.0.  
+
+#### General  
+
+> [!IMPORTANT]
+>  **Breaking Change**: WebView2 pre-release package 0.9.430 has been deprecated, and will be removed the upcoming release. Please discontinue development with this packages.  
+
+###### Features  
+*   Disabled Edge Shopping feature in WebView2. 
+*   Added [TrySuspend & Resume][ReferenceWin32Icorewebview210774PreReleaseTrySuspendResume] method that lets you suspend and resume a WebView. 
+*   Added [SetVirtualHostNameToFolderMapping][ReferenceWin32Icorewebview210774PreReleaseSetVirtualHostNameToFolderMapping] method that maps between a virtual host name and a folder path. 
+*   Added [DefaultBackgroundColor][ReferenceWin32Icorewebview2controllerViewWebview210774PreReleaseDefaultBackgroundColor] property that lets you set the background color and transparency.   \([\#414][GithubMicrosoftedgeWebviewfeedbackIssue549]\)
```

Comment:
Versions and link definitions to be added/updated.
```suggestion
*   Added [DefaultBackgroundColor][ReferenceWin32Icorewebview2controllerViewWebview210774PreReleaseDefaultBackgroundColor] property that lets you set the background color and transparency.   \([\#414][GithubMicrosoftedgeWebviewfeedbackIssue414]\)
```

Reply:
`GithubMicrosoftedgeWebviewfeedbackIssue414` is undefined.

Reply:
https://github.com/MicrosoftEdge/WebView2Feedback/issues/414 it works for me?